libraries/AP_Compass/CompassCalibrator.cpp

```diff
@@ -219,28 +234,32 @@ void CompassCalibrator::update(bool &failure) {
 /////////////////////////////////////////////////////////////
 ////////////////////// PRIVATE METHODS //////////////////////
 /////////////////////////////////////////////////////////////
-bool CompassCalibrator::running() const {
+bool CompassCalibrator::running() const
+{
     return _status == COMPASS_CAL_RUNNING_STEP_ONE || _status == COMPASS_CAL_RUNNING_STEP_TWO;
 }
 
-bool CompassCalibrator::fitting() const {
-    return running() && _samples_collected == COMPASS_CAL_NUM_SAMPLES;
+bool CompassCalibrator::fitting() const
+{
+    return running() && (_samples_collected == COMPASS_CAL_NUM_SAMPLES);
 }
 
-void CompassCalibrator::initialize_fit() {
```

Comment:
I think we need to discuss this style change at a devcall.

I'm really not fussed either way, but some team members strongly prefer the brace-on-same-line approach.

... and I don't want to be reviewing PRs where we move backwards-and-forwards between styles!

Reply:
Really?  I think the vast majority of our code does it this way in the .cpp files.  Anyway, I've added the Dev Call Topic label.